cryptoconditions/types/ed25519.py

```diff
@@ -66,7 +66,7 @@ def sign(self, message, private_key):
         #   .update(Buffer.concat([this.messagePrefix, this.message]))
         #   .digest()
 
-        self.signature = sk.sign(message, encoding=None)
+        self.signature = sk.sign(message, encoding='bytes')
```

Comment:
from the docstring: `message (string): message to be signed`
needs to be:

`message (bytes): message to be signed`